.codedeploy/scripts/add_node_specific_config

```diff
@@ -23,10 +23,10 @@ FLAGS=
 
 case "$NAME" in
 	vln)
-		FLAGS="--ws-port 80 --rpc-port 8080 --rpc-external --ws-external --rpc-cors all --node-key $VLN1_KEY"
+		FLAGS="--collator --ws-port 80 --rpc-port 8080 --rpc-external --ws-external --rpc-cors all -- --execution wasm --chain rococo"
 		NAME="node"
 		;;
-	*) FLAGS="--validator $(bootnodes vln)";;
+	*) FLAGS="$(bootnodes vln)";;
```

Comment:
Collator stuff needs to go here, the `*)` matches `odin`, `thor` and `loki` our former validators now collators.

Reply:
If you see that there are new flags that apply to both the full node and the collators we can add those in the `app.service` file.